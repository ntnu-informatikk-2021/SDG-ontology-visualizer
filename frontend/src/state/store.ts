import { createStore } from 'redux';
import apiErrorReducer from './reducers/apiErrorReducer';

const store = createStore(apiErrorReducer);

export default store;
