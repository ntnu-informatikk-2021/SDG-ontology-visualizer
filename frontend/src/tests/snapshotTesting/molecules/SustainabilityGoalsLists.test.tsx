import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import SustainabilityGoalsLists from '../../../components/molecules/SustainabilityGoalsList';
import store from '../../../state/store';

it('renders when there are no items', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <SustainabilityGoalsLists />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
