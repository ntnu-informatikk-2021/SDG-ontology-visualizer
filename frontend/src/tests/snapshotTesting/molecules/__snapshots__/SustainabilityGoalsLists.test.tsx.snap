// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<div
  className="chakra-stack css-2n1jg2"
>
  <div
    className="css-x75vte"
  />
</div>
`;
